src/sdk/pynni/nni/nas/pytorch/callbacks.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import os
+
+_logger = logging.getLogger(__name__)
+
+
+class Callback:
+
+    def __init__(self):
+        self.model = None
+        self.mutator = None
+        self.trainer = None
+
+    def build(self, model, mutator, trainer):
+        self.model = model
```

Comment:
Move the parameters to `__init__` would make the class easier to use.

Reply:
When callback is instantiated, mutator and trainer might not exist. So an extra build is needed.